src/helpers/highlight.ts

```diff
@@ -4,7 +4,8 @@ import { TAG_REPLACEMENT } from '../lib/escape-highlight';
 import { component } from '../lib/suit';
 
 export type HighlightOptions = {
-  attribute: string;
+  // @MAJOR only accept array of paths here
+  attribute: string | Array<string | number>;
```

Comment:
Since it's a path it makes sense to _only_ accept strings:

```
attribute: ['categories', '1'],
```

It'll make the type less permissive.